### PR TITLE
Projectile

### PR DESCRIPTION
**Pull request by raoulkent**

### CODE REVIEW

File: game/core/src/main/java/com/gg/game/model/Decor.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.gg.game.model;
+
+import com.gg.game.model.math.Rectangle;
+import com.gg.game.model.utils.Point;
+
+class Decor {
```
Comment (Riddarvid) [inline game/core/src/main/java/com/gg/game/model/Decor.java]: Is this class supposed to be part of this commit?

File: game/core/src/main/java/com/gg/game/model/Projectile.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
+package com.gg.game.model;
+
+import com.gg.game.model.utils.Point;
+import com.gg.game.model.utils.Vector;
+
+/**
+ * Represents a projectile in the world.
+ */
+public class Projectile implements Movable {
```
Comment (pudkipz) [inline game/core/src/main/java/com/gg/game/model/Projectile.java]: Shouldn't Projectile also have a collisionBox?

Comment (fourtooths) [reply game/core/src/main/java/com/gg/game/model/Projectile.java]: tänker också på detta^^, annars nice!

Comment (raoulkent) [reply game/core/src/main/java/com/gg/game/model/Projectile.java]: Whoopsie. Yep, that's why it was built wasn't it...

**Pull request merged**